lualib/resty/kong/log.lua

```diff
@@ -22,58 +22,56 @@ base.allows_subsystem("http")
 
 
 ffi.cdef([[
-int ngx_http_lua_kong_ffi_set_log_level(ngx_http_request_t *r, int level);
-]])
+int32_t
```

Comment:
I checked some code in this repo and lua-resty-core,  I think the return value type of ffi function should be `int`, it may be a better choice.

Reply:
need more discussions for it.